.github/workflows/ci.yaml

```diff
@@ -293,10 +294,6 @@ jobs:
       - name: create realm users
         run: pnpm register-realm-users
         working-directory: packages/matrix
-      # TODO: remove with CS-8104
-      - name: Install watchexec
-        run: cargo install --locked watchexec-cli
-        if: ${{ matrix.testModule == 'realm-endpoints-test.ts' }}
```

Comment:
I found that the test-hanging problem I originally added this workaround for didn’t happen when I used `node` as the `sane` file-watcher instead of `watchexec`, so this is no longer needed. The `testFileWatcher` is still in use though, so 8104 stays open.